lib/broccoli/ember-app.js

```diff
@@ -874,7 +875,7 @@ EmberApp.prototype.import = function(asset, options) {
     return;
   }
 
```

Comment:
Done. 

Tiny caveat, if users are adding paths that include whitespaces and try to cleverly escape them then this will break. Node is capable of handling the following:
```
app.import('bower_components/some lib/some file.js');
``` 
But this will break:
```
app.import('bower_components/some\ lib/some\ file.js');
``` 
I doubt that anyone is doing this but food for thought :information_desk_person: